products/iam/api.yaml

```diff
@@ -195,4 +198,236 @@ objects:
       - !ruby/object:Api::Type::Boolean
         name: 'deleted'
         description: The current deleted state of the role
-        output: true
\ No newline at end of file
+        output: true
+  - !ruby/object:Api::Resource
+    name: 'WorkloadIdentityPool'
```

Comment:
There are `ansible.yaml` and `inspec.yaml` files in this directory. Do you mind adding entries to them similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/iam/ansible.yaml#L47-L48 for this resource that mark it as `exclude: true`?

There is an unfortunate opt-in behaviour for new resources in a product, and they have required fields that cause our CI to report a failure when unspecified.

Reply:
I've added excludes for WorkloadIdentityPool in both ansible (datasource & overrides) and inspec.
I've also added the manually written resources (Role, ServiceAccount, ServiceAccountKey and OrganizationCustomRole) to the exclude list in terraform.yaml to prevent these resources from being generated

Reply:
Thanks!